services/route_manager/src/main/java/com/futurewei/alcor/route/controller/RouterController.java

```diff
@@ -315,7 +315,7 @@ public RouteTableWebJson getVpcRouteTableById(@PathVariable String projectid, @P
             method = GET,
             value = {"/project/{projectid}/subnets/{subnetid}/routetable"})
     @DurationStatistics
-    public RouteTableWebJson getOrCreateSubnetRouteTable(@PathVariable String projectid, @PathVariable String subnetid) throws Exception {
+    public RouteTableWebJson getSubnetRouteTable(@PathVariable String projectid, @PathVariable String subnetid) throws Exception {
```

Comment:
@kevin-zhonghao I think @DavidLiu506 also modify this function, please check his PR#655